.phpcs.xml

```diff
@@ -2,5 +2,10 @@
 <ruleset>
 	<rule ref="./vendor/mediawiki/mediawiki-codesniffer/MediaWiki">
 	</rule>
+
+	<rule ref="MediaWiki.Commenting.FunctionComment">
```

Comment:
Was this to make it pass sniffing for now instead of fixing it? Not sure i get what MediaWiki.Commenting.FunctionComment is referring to exactly.

Reply:
> Ignore the PHPCS FunctionComment sniff, since functions can be fully
self-documenting using static parameter and return types.

In other words, PHPCS wants a comment like

```php
/*
 * @param ContainerInterface|null $services
 * @return ExternalLinks
 */
public static function getExternalLinks( ContainerInterface $services = null ): ExternalLinks {
```

and in my opinion that’s just pointless. (We ignore the sniff in Wikibase for mostly the same reason.)

Reply:
yeah that sounds a bit too strict